### PR TITLE
Retire rmfitzpatrick

### PR DESCRIPTION
**Pull request by rmfitzpatrick**
👋 I've been absent from this project and my role at Splunk is pretty much exclusively related to collector items for the foreseeable future. I do intend to help with the metric gatherer and its improvement as I can, but have obviously not been contributing as a maintainer for some time.

Thanks to @trask, @mateuszrzeszutek, and all the other members for greatly improving this project and keeping it cruising.

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]
Thanks for all the work @rmfitzpatrick !

File: README.md [hunk 1]
```diff
@@ -58,9 +58,13 @@ Maintainers ([@open-telemetry/java-contrib-maintainers](https://github.com/orgs/
 - [Jack Berg](https://github.com/jack-berg), New Relic
 - [Mateusz Rzeszutek](https://github.com/mateuszrzeszutek), Splunk
 - [Nikita Salnikov-Tarnovski](https://github.com/iNikem), Splunk
-- [Ryan Fitzpatrick](https://github.com/rmfitzpatrick), Splunk
 - [Trask Stalnaker](https://github.com/trask), Microsoft
 
+Emeritus maintainers:
+
+- [Anuraag Agrawal](https://github.com/anuraaga), AWS
```
Comment (trask) [inline README.md:65]: @anuraaga requested not to be listed here
```suggestion
```

**Comment by trask** [comment]
closing and re-opening to unstick CI

**Comment by trask** [comment]
thx @rmfitzpatrick for founding the contrib repo!

**Pull request merged**